Afternoon Projects/Unit 1 - UIKit & Fundamentals/Day 1/Flashlight-master/README.md

```diff
@@ -1 +1,59 @@
-Test README addition
+# Flashlight Readme
+# FlashLight
+---
+Students will build a simple Flashlight app to practice working with IBOutlets, IBActions, and UIControlEvents.
+Students who complete this project independently are able to:
+* Use, understand, and describe different UIControls available in Cocoa Touch
+* Use, understand, and describe UIControlEvents
+* Create and use IBOutlets to get access to Storyboard elements in code
+* Create and use IBActions to run code on a UIControlEvent
+* Work  through the logic required for a flashlights functions
+
+
+## Guide
+---
+### Storyboard Setup
+
+Build the initial view a user will see. This view will have two states that we will control programmatically. The first state has a black background and an ‘Turn On’ button with white text. When the user taps ‘On’ change the background to white, update the button’s title to ‘Turn Off’, and change the button text color to black. The button should toggle the two states.
+
+### FlashlightViewController
+
+1. Delete the initial ViewController.Swift file
+2. Create a new Cocoa Touch Class File named `FlashlightViewController`
+3. Subclass the curated ViewController on your Main.storyboard to a `FlashlightViewControler`
+4. Change the background color of the view to Black
+5. Build and Run your app and check for bugs. The app should display a black screen.
+6. Navigate to the `Object Library` and drag out a `Button`
+7. Control-Drag from the Button to your view. With your shift key held - select Center Horizontally in Container and Center Vertically in Container.
+8. Tap the Add Constraints icon ( Tie Fighter) and set the `Width` to 200 and `Height`to 100
+9. Navigate to the Attributes inspector and set your font size to `System 30`
+10. Set your default title to `Turn On`
+11. Set your text color to `White Color`
+12. Build and Run your app and check for bugs. The app should display a black screen, with a button displaying Turn On in white text
+  
+### Outlets / Actions
+
+1. Navigate to your Main.storyboard file and select your Flashlight ViewController 
+2. Click on your assistant editor. It looks like two rings that slightly intersect.
+3. Create an `Outlet` for your button by control-draging from your `Button` to the FlashlightViewControler.swift file that your Assistant editor opened. Name your button `flashlightToggleButton`
+** Remember: Outlets should be placed above your `ViewDidLoad`
+
+5. Create an `isOn`  `Bool` that you will use to track the buttons state. Set the default value to `false`
```

Comment:
Maybe include a hint? Not sure if this is really needed or not for week 1 day 1


  * <details>
        <summary> <code> Hint </code> </summary>

        var isOn: Bool = false
   </details>